.travis.yml

```diff
@@ -10,5 +10,5 @@ before_install:
 script:
   - go test -v ./...
 after_success:
-  - gox -output "dist/{{.OS}}_{{.Arch}}_{{.Dir}}"
-  - ghr --username iangrunert --token $GITHUB_TOKEN --replace --prerelease --debug pre-release dist/
\ No newline at end of file
+  - scripts/build.sh
```

Comment:
But if you click on line 489 it will expand to:

```
3.04s$ scripts/build.sh
Number of parallel builds: 2
-->       linux/386: github.com/iangrunert/git-ratchet
-->      darwin/386: github.com/iangrunert/git-ratchet
-->     linux/amd64: github.com/iangrunert/git-ratchet
-->       linux/arm: github.com/iangrunert/git-ratchet
-->    darwin/amd64: github.com/iangrunert/git-ratchet
-->     windows/386: github.com/iangrunert/git-ratchet
-->   windows/amd64: github.com/iangrunert/git-ratchet
==> Results:
pkg/darwin_386:
total 3.6M
-rwxr-xr-x 1 travis travis 3.6M Jul 10 03:54 git-ratchet
pkg/darwin_amd64:
total 4.5M
-rwxr-xr-x 1 travis travis 4.5M Jul 10 03:54 git-ratchet
pkg/linux_386:
total 3.7M
-rwxr-xr-x 1 travis travis 3.7M Jul 10 03:54 git-ratchet
pkg/linux_amd64:
total 4.6M
-rwxr-xr-x 1 travis travis 4.6M Jul 10 03:54 git-ratchet
pkg/linux_arm:
total 3.7M
-rwxr-xr-x 1 travis travis 3.7M Jul 10 03:54 git-ratchet
pkg/windows_386:
total 3.8M
-rwxr-xr-x 1 travis travis 3.8M Jul 10 03:54 git-ratchet.exe
pkg/windows_amd64:
total 4.7M
-rwxr-xr-x 1 travis travis 4.7M Jul 10 03:54 git-ratchet.exe
```